lib/KBox/src/KMessage.h

```diff
@@ -191,6 +196,94 @@ class IMUMessage: public KMessage {
     };
 };
 
+class NAVMessage: public KMessage {
+  private:
+    bool apMode, apHeadingMode, apWaypointMode, apDodgeMode;
+    double currentHeading, targetHeading, courseToWaypoint;
```

Comment:
I don't think the `AutopilotTask` should rely on `currentHeading` being provided by the `NAVMessage`. If the currently displayed page on KBox is not the navigation page, then those messages will not be sent and the autopilot will not get updates to the heading.

I think that instead, the autopilot should listen to `IMUMessage` and update its currentHeading based on IMUMessages.

Do you agree? If yes, I can do it - or you can do it. Tell me what you prefer.

Also, I don't think there is a difference between targetHeading and courseToWaypoint. I think they are both the same. As far as I can tell, `courseToWaypoint` is not used now and we should remove it so it's easier to read and understand the code.